src/schema/validator.js

```diff
@@ -69,17 +72,47 @@ const _validateLangPack = validator.compile({
   id: 'langpack-manifest',
   $ref: '#/types/WebExtensionLangpackManifest',
 });
+
 export const validateLangPack = (...args) => {
   const isValid = _validateLangPack(...args);
   validateLangPack.errors = filterErrors(_validateLangPack.errors);
   return isValid;
 };
 
+
+const _validateStaticTheme = validator.compile({
+  ...merge(
```

Comment:
Nit, An inline comment that explains "what we are doing here and why" could be helpful (e.g. why we merge these object and why we make a new object with a copy of the properties of the resulting object here).